config/runtime/tests/fluster-chromeos.jinja2

```diff
@@ -66,4 +74,5 @@
               -i /home/cros/.ssh/id_rsa
               root@$(lava-target-ip)
               poweroff && sleep 30 || true
-            - python3 /usr/bin/fluster_parser.py --results
+            - apt install -y python3-junitparser
```

Comment:
Please have this package installed directly in the docker image. Otherwise this will run for each job, wasting both time and bandwidth.